tests/testthat/test-txt_wrap.R

```diff
@@ -150,16 +152,133 @@ test_that("row label wrapping has identical indentation", {
   mf_rinfo(matform)$indent <- c(1, 2)
   matform$strings[2, 1] <- paste0(" ", matform$strings[2, 1]) # Does not respect indent_size
   matform$strings[3, 1] <- paste0("  ", matform$strings[3, 1]) # Does not respect indent_size
+  expect_error(
+    catform <- toString(matform, widths = c(15, 5), hsep = "-"), # Broken indentation, check fails
+    "indentation mismatches"
+  )
+
+  matform$strings[2, 1] <- paste0("  ", matform$strings[2, 1]) # Does not respect indent_size
+  matform$strings[3, 1] <- paste0("    ", matform$strings[3, 1]) # Does not respect indent_size
   catform <- toString(matform, widths = c(15, 5), hsep = "-") # This reindent (correctly) the rows
   res_vec <- strsplit(catform, "\n")[[1]]
   exp_vec <- c(
     "                   all_o",
     "                    bs  ",
     " -----------------------",
-    "   Something to    3    ",
+    "    Something to   3    ",
     "   wrap                 ",
-    "     Also here     4    ",
+    "       Also here   4    ",
     "     it is              "
```

Comment:
I think here I added too many spaces in the fake indentation. It seems to handle well the additional spaces and preserves them